src/makeChange.js

```diff
@@ -1,3 +1,37 @@
 export default function makeChange({price, amountGiven}) {
-  // your code here
+var object = {'quarters':0 , 'dimes':0 , 'nickels':0 , 'pennies':0 }
+var difference = amountGiven - price
+  if  (difference >= 25) {
+    // # of quarters
+    Math.floor(difference / 25);
+    var quarters = Math.floor(difference / 25);
```

Comment:
the only other place you use this variable is to set object.quarters. In cases like this you dont really need a variable. you can just set object.quarters = Math.floor(difference % 25)